lib/benchee/formatters/csv.ex

```diff
@@ -90,38 +83,43 @@ defmodule Benchee.Formatters.CSV do
       ...> }
       iex> suite
       iex> |> Benchee.Formatters.CSV.format
-      iex> |> Map.get("Some Input")
       iex> |> Enum.take(2)
-      ["Name,Iterations per Second,Average,Standard Deviation,Standard Deviation Iterations Per Second,Standard Deviation Ratio,Median,Minimum,Maximum,Sample Size\\r\\n",
-       "My Job,2.0e3,500.0,200.0,800.0,0.4,450.0,200,900,8\\r\\n"]
+      ["Name,Input,Iterations per Second,Average,Standard Deviation,Standard Deviation Iterations Per Second,Standard Deviation Ratio,Median,Minimum,Maximum,Sample Size\\r\\n",
+       "My Job,Some Input,2.0e3,500.0,200.0,800.0,0.4,450.0,200,900,8\\r\\n"]
 
   """
   @spec format(Suite.t) :: %{Suite.key => String.t}
```

Comment:
The CI failures seem to be pointing to this typespec now being out of date. I think once it's updated then we should be good to go!